### PR TITLE
Add io.gitlab.elescoute.password

### PR DESCRIPTION
**Pull request by elescoute**
### Password

Password is a password calculator and random generator for Gnome written in Gtk/Vala.
This app calculates strong unique passwords for each alias and passphrase combination.
No need to remember dozens of passwords any longer and no need for a password manager!
The calculator can use MD5, SHA1, SHA256 and SHA512 hash algorithms in combination with HMAC.
Compatible with Linux Phone using Phosh (PinePhone, Librem 5).

### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: io.gitlab.elescoute.password.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    "release": "1.6.0",
+                    "commit": "733312921f9cdeaa0973f4cf565357a903d50625"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {
+        }
+    }
```
Comment (hfiguiere) [inline io.gitlab.elescoute.password.json:44]:
```suggestion
    ]
```

**Comment by hfiguiere** [comment]
bot, build io.gitlab.elescoute.password

**Comment by elescoute** [comment]
Metainfo fixed!

**Comment by elescoute** [comment]
Ready to build!

**Comment by hfiguiere** [comment]
you can also

bot, build io.gitlab.elescoute.password

**Comment by hfiguiere** [comment]
in the appstream file

```
	<id>io.gitlab.elescoute.password.desktop</id>
```

This is incorrect. id should NOT end in .desktop.

```
  <project_group>GNOME</project_group>
```

This is not allowed as it is not a GNOME official app.

Also the 
```
     <value key="Purism::form_factor">workstation</value>
``` 

should be removed

**Comment by elescoute** [comment]
bot, build io.gitlab.elescoute.password

**Comment by hfiguiere** [comment]
> in the appstream file
> 
> ```
> 	<id>io.gitlab.elescoute.password.desktop</id>
> ```
> 
> This is incorrect. id should NOT end in .desktop.
> 

It appear that this was ignored. Hence the build failure.

Whatever template / documentation said to add `.desktop` is wrong. Had been for years.

**Comment by elescoute** [comment]
bot, build io.gitlab.elescoute.password

**Comment by elescoute** [comment]
Tested successfully!

**Pull request closed without merging**